cytosnake/cli/exec/workflow_exec.py

```diff
@@ -38,13 +39,31 @@ def __base_exec(
     -------
     bool
         True for successful execution, otherwise False
+
+    Raises
+    ------
+    TypeError
+        Raised if env_manager is not a string type
```

Comment:
Consider including `ValueError` here as well to reflect the error raised if `env_manager` is not conda or mamba.